Firebase/Messaging/FIRMessagingUtilities.m

```diff
@@ -149,8 +149,24 @@ int64_t FIRMessagingCurrentTimestampInMilliseconds(void) {
   return version;
 }
 
+
+NSString * FIRMessagingBundleIdentifierByRemovingLastPartFrom(NSString *bundleIdentifier) {
+  NSString *bundleIDComponentsSeparator = @".";
+
+  NSMutableArray<NSString *> *bundleIDComponents =
+      [[bundleIdentifier componentsSeparatedByString:bundleIDComponentsSeparator] mutableCopy];
+  [bundleIDComponents removeLastObject];
+
+  return [bundleIDComponents componentsJoinedByString:bundleIDComponentsSeparator];
+}
+
 NSString *FIRMessagingAppIdentifier(void) {
-  return [[NSBundle mainBundle] bundleIdentifier];
+  NSString *bundleIdentifier = [[NSBundle mainBundle] bundleIdentifier];
```

Comment:
Nit: `s/bundleIdentifier/bundleID/` for the variable name.

Reply:
Done.